.github/workflows/tests.yaml

```diff
@@ -328,6 +328,27 @@ jobs:
           nox -f tests/integration/goldens/eventarc/noxfile.py -s mypy-3.12 blacken lint
           nox -f tests/integration/goldens/logging/noxfile.py -s mypy-3.12 blacken lint
           nox -f tests/integration/goldens/redis/noxfile.py -s mypy-3.12 blacken lint
+  goldens-unit:
```

Comment:
I think we were using `3.12` as the latest supported Python version. If that's the case, should we be using `3.13` here?

Reply:
Sounds good. I'll update this to Python 3.13 in PR https://github.com/googleapis/gapic-generator-python/pull/2215